.github/workflows/ci.yml

```
@@ -63,9 +87,9 @@ jobs:
               owner: context.repo.owner,
               repo: context.repo.repo,
               issue_number: prNumber,
-              body: `### 빌드 결과\n${message}`
+              body: `### 빌드 및 린트 결과\n${message}`
             });
 
-      - name: Fail job if build failed
-        if: steps.build.outcome == 'failure'
+      - name: Fail job if any step failed
+        if: steps.check.outcome == 'failure' || steps.build.outcome == 'failure'
```

actionlint (1.7.7)

property "check" is not defined in object type {build: {conclusion: string; outcome: string; outputs: {string => string}}; lint: {conclusion: string; outcome: string; outputs: {string => string}}; status: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)